scaffolds/ruby/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM ruby:3.0
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
```

I had this error when trying to build this Dockerfile:

```
Step 5/5 : RUN bundle install
 ---> Running in 638a3d385e92
/usr/local/lib/ruby/3.0.0/rubygems.rb:281:in `find_spec_for_exe': Could not find 'bundler' (1.17.1) required by your /usr/src/app/Gemfile.lock. (Gem::GemNotFoundException)
To update to the latest version installed on your system, run `bundle update --bundler`.
To install the missing version, run `gem install bundler:1.17.1`
	from /usr/local/lib/ruby/3.0.0/rubygems.rb:300:in `activate_bin_path'
	from /usr/local/bin/bundle:23:in `<main>'

```

This change fixed it :

```suggestion
RUN bundle config --global frozen 1 && gem install bundler:1.17.1
```

---

Interesting, I do not encounter this error on my computer :thinking: Let's commit your changes!